opencti-platform/opencti-graphql/src/modules/publicDashboard/publicDashboard-domain.ts

```diff
@@ -125,6 +126,20 @@ export const addPublicDashboard = async (
     throw FunctionalError('Cannot published empty dashboard');
   }
 
+  const adminAuthorizedMembers = dashboard.authorized_members
+    .filter((member) => member.access_right === 'admin')
+    .map((member) => member.id);
+
+  if (!adminAuthorizedMembers.includes(user.id)) {
```

Comment:
what about when you are authorized as part of a group ? there are already dedicated functions in access.ts for authorized members. You can use : 

```suggestion
const access = getUserAccessRight(user, dashboard);
if (access !== MEMBER_ACCESS_RIGHT_ADMIN) {
     throw ForbiddenAccess('You are not allowed to do this.');
}
```